src/nextjs/nextjs-wizard.ts

```diff
@@ -329,12 +329,13 @@ export async function runNextjsWizardWithTelemetry(
     path.join(process.cwd(), 'vercel.json'),
   );
 
-  if (mightBeUsingVercel) {
+  if (mightBeUsingVercel && !options.comingFrom) {
     clack.log.info(
-      "▲ It seems like you're using Vercel. We recommend using the Sentry Vercel integration to set up an auth token for Vercel deployments: https://vercel.com/integrations/sentry",
+      "▲ It seems like you're using Vercel. We recommend using the Sentry Vercel \
+      integration to set up an auth token for Vercel deployments: https://vercel.com/integrations/sentry",
     );
   } else {
-    await traceStep('configure-ci', () => configureCI('nextjs', authToken));
+    await setupCI('nextjs', authToken, options.comingFrom);
```

Comment:
random thought (feel free to ignore/PR separately): We could ask users if we should open their browser and use `opn` to do that 🤔